serverless.yml

```diff
@@ -10,23 +10,32 @@ provider:
 package:
   excludeDevDependencies: true
   individually: true
-  patterns:
-    - '!./**'
+  exclude:
+    - ./**
+
 
 functions:
   HousingRepairsOnlineFrontend:
     name: ${self:service}-${self:provider.stage}
     handler: lambda.handler
     timeout: 30
     package:
-      patterns:
-        - 'lambda.js'
-        - 'next.config.js'
-        - 'build/_next/**'
-        - 'public/**'
-        - 'node_modules/**'
-        - '!node_modules/@next/swc-darwin-arm64'       
-        - '!node_modules/@sentry/cli/sentry-cli'         
+    
+      include:
+        - lambda.js
+        - next.config.js
+        - public/**
+        - 'build/_next/BUILD_ID'
+        - 'build/_next/*.json'
+        - 'build/_next/server/**'
+        - 'build/_next/static/**'
+        - node_modules/**
+        - '!node_modules/@next/swc-darwin-arm64'
+        - '!node_modules/@next/swc-darwin-x64'
+        - '!node_modules/@next/swc-linux-x64-gnu'
+        - '!node_modules/@next/swc-win32-x64-msvc'
+        - '!node_modules/@sentry/cli/sentry-cli'
+        - '!node_modules/leaflet'    

```

Comment:
How do you know which modules to exclude?
Like, how do you know you don't need: `swc-darwin` or `leaflet`?

Reply:
im just that smart